java/HealthVault Mobile/HVSample/src/main/java/com/microsoft/hsg/android/hvsample/MainActivity.java

```diff
@@ -0,0 +1,274 @@
+package com.microsoft.hsg.android.hvsample;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import android.app.ListActivity;
+import android.app.ProgressDialog;
+import android.content.Context;
+import android.content.Intent;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.View.OnClickListener;
+import android.widget.ArrayAdapter;
+import android.widget.LinearLayout;
+import android.widget.ListView;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+import android.widget.Toast;
+import android.support.design.widget.BottomNavigationView;
+
+import com.microsoft.hsg.HVException;
+import com.microsoft.hsg.android.simplexml.HealthVaultApp;
+import com.microsoft.hsg.android.simplexml.HealthVaultInitializationHandler;
+import com.microsoft.hsg.android.simplexml.HealthVaultSettings;
+import com.microsoft.hsg.android.simplexml.client.HealthVaultClient;
+import com.microsoft.hsg.android.simplexml.client.RequestCallback;
+import com.microsoft.hsg.android.simplexml.things.types.types.PersonInfo;
+import com.microsoft.hsg.android.simplexml.things.types.types.Record;
+
+public class MainActivity
+	extends ListActivity
+	implements HealthVaultInitializationHandler {
+
+	private HealthVaultApp mService;
+	private HealthVaultClient mHVClient;
+	private BottomNavigationView mBottomNav;
+	private LinearLayout mWeightTile;
+
+	private ProgressDialog connectProgressDialog;
+
+	@Override
+	protected void onCreate(Bundle savedInstanceState) {
+		super.onCreate(savedInstanceState);
+		setContentView(R.layout.activity_main);
+
+		mService = HealthVaultApp.getInstance(this);
+		if (mService.isAppConnected()) {
+			mService.start(this, this);
+		}
+
+		mHVClient = new HealthVaultClient();
+		
+		mBottomNav = (BottomNavigationView) findViewById(R.id.navigation);
+
+		mBottomNav.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
```

Comment:
This listener does nothing, is it in here as an example? if not we can just remove it, and remove the lookup of the control above